presto-testing/src/main/java/io/prestosql/testing/DistributedQueryRunner.java

```diff
@@ -111,14 +114,26 @@ private DistributedQueryRunner(
             ImmutableList.Builder<TestingPrestoServer> servers = ImmutableList.builder();
 
             for (int i = 1; i < nodeCount; i++) {
-                TestingPrestoServer worker = closer.register(createTestingPrestoServer(discoveryServer.getBaseUrl(), false, extraProperties, environment, baseDataDir));
+                TestingPrestoServer worker = closer.register(createTestingPrestoServer(
+                        discoveryServer.getBaseUrl(),
+                        false,
+                        extraProperties,
+                        environment,
+                        baseDataDir,
+                        systemAccessControl));
```

Comment:
don't configure SAC on worker
people know it's not required and they don't do this; so we shouldn't do this in tests either